src/app/components/listing/confirm/confirm-developer.jsx

```diff
@@ -232,157 +235,159 @@ function ChplConfirmDeveloper(props) {
   });
 
   return (
-    <Container maxWidth="md">
-      <div className={classes.developerConfirm}>
-        <div className={classes.developerSubContainer}>
-          <Button
-            variant="outlined"
-            color="default"
-            fullWidth
-            className={`${classes.buttonCard} ${!isCreating ? classes.buttonCardFocused : ''}`}
-            onClick={() => handleCreationToggle(false)}
-          >
-            <span className={classes.buttonContent}>
-              <CheckCircleIcon color="primary" className={classes.extraLargeIcons} />
-              { selectedDeveloper
-                ? (
-                  <>
-                    {`Use "${selectedDeveloper.name}"`}
-                  </>
-                ) : (
-                  <>
-                    Choose A Developer To Use
-                  </>
-                )}
-            </span>
-          </Button>
-          <div className={classes.orContainer}>
-            <Divider />
-            <Typography>OR</Typography>
-            <Divider />
+    <div className={classes.fixFooterSpacing}>
+      <Container maxWidth="md">
```

Comment:
There's a component down below that has the className inside the Container tag:
```suggestion
      <Container maxWidth="md" className={classes.fixFooterSpacing}>
```
which means that the extra `div` isn't needed, which means that all of the indentation changes in this file probably aren't needed either